### PR TITLE
refactor: 💡 use nest to run swagger generation instead of jest

### PR DESCRIPTION
**Pull request by polymath-eric**
Jest is meant to be a test runner, and not a general purpose task runner.

I think its cleaner to use just NestJS.

I tried getting it to write to stdout, but extra stuff got added in. the `logger` false keep it pretty quite though.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # compiled output
 /dist
 /node_modules
+polymesh-rest-api-swagger-spec.json
```
Comment (polymath-eric) [inline .gitignore:4]: This maybe could go in `/dist`, but not sure how that affects the pipe line.

**Review by prashantasdeveloper — APPROVED** [review]

File: src/developer-testing/developer-testing.module.ts [hunk 2]
```diff
@@ -14,8 +14,7 @@ export class DeveloperTestingModule {
   static register(): DynamicModule {
     const controllers = [];
 
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const DEVELOPER_UTILS: boolean = JSON.parse(process.env.DEVELOPER_UTILS!);
+    const DEVELOPER_UTILS: boolean = JSON.parse(`${!!process.env.DEVELOPER_UTILS}`);
```
Comment (polymath-eric) [inline src/developer-testing/developer-testing.module.ts:17]:
```ts
const DEVELOPER_UTILS: boolean = JSON.parse(process.env.DEVELOPER_UTILS!);
                                          ^
SyntaxError: Unexpected token u in JSON at position 0
```

TypeScript was right again

**Comment by polymath-eric** [comment]
seems like I war relying on having a .env defined. working on the errors

File: .gitignore (hunk 1 above)
Comment (polymath-eric) [reply .gitignore:4]: This should NOT go into `/dist` its ~300KB and would be bloat in an image / package.

**Pull request merged**